src/app/accounts/lock.component.html

```diff
@@ -1,10 +1,10 @@
 <form id="lock-page" (ngSubmit)="submit()">
     <div class="content">
         <p aria-hidden="true"><i class="fa fa-lock fa-4x text-muted"></i></p>
-        <p>{{(pinLock ? 'yourVaultIsLockedPinCode' : 'yourVaultIsLocked') | i18n}}</p>
+        <p>{{'yourVaultIsLocked' | i18n}}</p>
```

Comment:
I think we should still show a different text when unlocking with PIN as it has no effect in regards to KeyConnector

Reply:
@Hinton Spoke to Danielle and she is fine with only having one label. Do you think we could simplify the structure for the biometrics button with regards to my comment above? If not, it's looking good for approval

Reply:
I think we could do that. Lets loop it by design.